app/.storybook/i18next.js

```diff
@@ -0,0 +1,30 @@
+// Configure i18next for storybook addon storybook-react-i18next
+// See https://storybook.js.org/addons/storybook-react-i18next
+import { initReactI18next } from 'react-i18next'
+import i18n from 'i18next'
+import Backend from 'i18next-http-backend'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+const ns = ['common']
+const supportedLngs = ['en', 'es']
```

Comment:
could this array could be imported from `next-i18next.config.js` to reduce needing to maintain it in multiple files?